htdocs/install/mysql/tables/llx_subscription.sql

```diff
@@ -24,7 +24,7 @@ create table llx_subscription
   fk_adherent     integer,
   fk_type         integer,
   dateadh         datetime,
-  datef           date,
+  datef           datetime,
```

Comment:
If you change date format into database, you must also change it into code (for exemple $field in class subscription)